src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -4782,6 +4782,8 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
         source = os_disk_image = data_disk_images = None
         if managed_image is not None:
             source = GalleryArtifactVersionFullSource(id=managed_image)
+        if virtual_machine is not None:
```

Comment:
The issue mentioned the min API version is 2023-07-03. Should we wrap the code with min_api check?
![image](https://github.com/Azure/azure-cli/assets/32201005/1ea34eed-5857-460c-8d40-3488ad4b6557)

Reply:
Yes,added.